app/controllers/plans/index.js

```diff
@@ -31,18 +40,18 @@ export default Controller.extend({
   ),
 
   showAnnual: true,
-  scrollToContact: false,
+  scrollToContact: equal('section', SECTION.CONTACT),
 
   actions: {
     signIn() {
       this.auth.signIn();
     },
 
     toggleContactScroll() {
-      this.set('scrollToContact', true);
-      setTimeout(
-        () => this.set('scrollToContact', false),
-        500
+      this.set('section', SECTION.NONE);
+      later(
+        () => this.set('section', SECTION.CONTACT),
+        100
```

Comment:
I'm curious, would it work with just `next`? Theoretically, it should, because `next` ensures that all values from the previous run are set and in effect.
```js
next(() => this.set('section', SECTION.CONTACT))
```

Reply:
Oh yeah, that probably would work. I'll try it out.

Reply:
Change made :)